t/t4018/php-enum

```diff
@@ -0,0 +1,3 @@
+enum RIGHT: string {
+    case Foo = 'ChangeMe';
```

Comment:
This seems to break the test suite: https://github.com/git/git/pull/1082/checks?check_run_id=3465586494#step:5:3769

Reply:
Fixed at 1fe4aa4e692c099e5d7098b1e4076482b7e18eb4 (squashed)